components/TimeSeriesCard.client.vue

```diff
@@ -31,47 +29,17 @@
 
 <script setup lang="ts">
 import { CIcon } from "@coreui/icons-vue";
-import Highcharts from "highcharts/esm/highcharts";
-import { useUniqueId } from "@coreui/vue";
+import useSynchroniseCharts from "~/composables/useChartSynchroniser";
```

Comment:
I agree that `useSynchroniseCharts` is better than `useChartSynchroniser` so should just update the file name.